### ISSUE TITLE
HogQL charts error on dashboards

### ISSUE THREAD

**Issue by isAdrisal** [PostHog/posthog#24177]
## Bug description

Example insight: [https://us.posthog.com/project/34169/insights/TpO43anx](https://us.posthog.com/project/34169/insights/TpO43anx)

When a HogQL insight chart is added to a dashboard, it shows Javascript error instead of the chart. Clicking through to the full insight page displays the chart as expected.

Example sentry entry id: `094c94414dcc49e9bddba2e2ba6d6a6b`

## How to reproduce

1. Create a HogQL insight and set the visualisation to any chart type
2. Add the insight to a dashboard
3. View the dashboard and see the chart error

## Additional context
<img width="1113" alt="Screenshot 2024-08-05 at 11 50 06" src="https://github.com/user-attachments/assets/7d75d95e-acc8-4bc4-b560-821b13f74894">
<img width="830" alt="Screenshot 2024-08-05 at 11 49 28" src="https://github.com/user-attachments/assets/9028e5dd-2e48-4b2f-80bb-27091541a51f">

### Debug info

```shell
Session: https://us.posthog.com/project/sTMFPsFhdP1Ssg/replay/01911fab-c3fb-7299-86ee-f83474fb8d16?t=9566
Admin: http://go/adminOrgUS/018a1aa0-fba1-0000-7c63-3525ad2656c8 (project ID 34169)
Sentry: http://go/sentryUS/34169
```

**Comment by MarconLP** [comment]
The issue is still there but instead of an error the insight is just blank
![2024-08-05 at 11 21 09@2x](https://github.com/user-attachments/assets/9cea9242-8379-4e51-a525-293151e66259)

**Comment by skoob13** [comment]
@isAdrisal thank you for reporting this issue. We fixed the problem today in #24183.
